src/test/kotlin/lotto/LottoTest.kt

```diff
@@ -0,0 +1,50 @@
+package lotto
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class LottoTest {
+
+    // checkAmountValidation
+    @ParameterizedTest
+    @ValueSource(strings = ["", "200", "-30"])
+    fun `구매금액 확인`(amount: String) {
```

Comment:
`@DisplayName`을 사용하여 테스트 함수의 의도를 표현하면 어떨까요?

```suggestion
    @DisplayName("구매 금액 확인")
    @ParameterizedTest
    @ValueSource(strings = ["", "200", "-30"])
    fun checkAmountValidation(amount: String) {
```